### ISSUE TITLE
[Feature Request] Make ncenter applets add/removeable in budgie-settings

### ISSUE THREAD

**Issue by Justinzobel** [ikeydoherty/budgie-next#8]
Start with a default selection of session controls, volume, notifications, calendar etc.

Possible options
mpris
rss reader
email client that lists newest mail (clicking on applet opens default mail client)
eopkg update checker
eopkg new package lister
weather

**Comment by JoshStrobl** [comment]
Given the Control Center already has a notification settings section, would it be viable to swap out the section "Notification Banners" label and use it as "Notification Center" instead, with the applications that have notifications enabled also then being able to leverage things like mpris (VLC), Weather would naturally have a widget and just enable / disable via Notifications section of Control Center, Terminal notifications, etc.

**Comment by Justinzobel** [comment]
Yeah whatever works really, I'm just throwing out ideas on possibly new features :)
